package.json

```diff
@@ -89,6 +89,7 @@
   "main": "dist/cytoscape.cjs.js",
   "unpkg": "dist/cytoscape.min.js",
   "jsdelivr": "dist/cytoscape.min.js",
+  "module": "dist/cytoscape.esm.js",
```

Comment:
Let's not include the `module` field until webpack issues are resolved.  The documentation could be updated to indicate that an alias needs to be used for ESM support with webpack or rollup.